tokio/src/process/mod.rs

```diff
@@ -672,6 +672,9 @@ impl Command {
     #[cfg(unix)]
     #[cfg_attr(docsrs, doc(cfg(unix)))]
     pub fn uid(&mut self, id: u32) -> &mut Command {
+        #[cfg(target_os = "nto")]
+        self.std.uid(id as i32);
+        #[cfg(not(target_os = "nto"))]
         self.std.uid(id);
```

Comment:
```suggestion
        #[cfg(target_os = "nto")]
        let id = id as u32;
        self.std.uid(id);
```

Reply:
Should be fixed in @AkhilTThomas  commit https://github.com/tokio-rs/tokio/pull/6421/commits/29cfcf827ce1964fcfac0b58fa8666c54f7c6917

Reply:
Please fix it everywhere.